### PR TITLE
Test/files facade tests

### PR DESCRIPTION
**Pull request by aince42**

### CODE REVIEW

File: packages/enmeshed_runtime_bridge/integration_test/test_assets/testFile.txt
Comment (jkoenig134) [inline packages/enmeshed_runtime_bridge/integration_test/test_assets/testFile.txt:1]: do we need an asset for that? We can also transform a string to bytes.

File: packages/enmeshed_runtime_bridge/pubspec.yaml [hunk 1]
```diff
@@ -36,6 +36,8 @@ flutter:
   assets:
     - assets/loki.js
     - assets/index.js
+    # Assets for testing
+    - integration_test/test_assets/testFile.txt
```
Comment (jkoenig134) [inline packages/enmeshed_runtime_bridge/pubspec.yaml:40]: when we remove the asset we should also remove this

File: packages/enmeshed_runtime_bridge/integration_test/services/facades/files_facade_test.dart [hunk 2]
```diff
@@ -0,0 +1,300 @@
...
+
+      final file = fileResult.value;
+      // globalFile = file;
+
+      expect(file, isInstanceOf<FileDTO>());
+      expect(file.filename, 'facades/test.txt');
+      expect(file.mimetype, 'plain');
+      expect(file.title, 'aTitle');
+    });
+    test('returns a valid FileDTO with all properties', () async {
```
Comment (jkoenig134) [inline packages/enmeshed_runtime_bridge/integration_test/services/facades/files_facade_test.dart:62]: formatting. newline between closing braces and new test.

File: packages/enmeshed_runtime_bridge/integration_test/services/facades/files_facade_test.dart [hunk 3]
```diff
@@ -0,0 +1,300 @@
...
+  group('FilesFacade: uploadOwnFile', () {
+    test('returns a valid FileDTO', () async {
+      final data = await rootBundle.load('integration_test/test_assets/testFile.txt');
+      final bytes = data.buffer.asUint8List().toList();
+
+      final fileResult = await session.transportServices.files.uploadOwnFile(
+        content: bytes,
+        filename: 'facades/test.txt',
+        mimetype: 'plain',
+        expiresAt: DateTime.now().add(const Duration(minutes: 5)).toRuntimeIsoString(),
```
Comment (jkoenig134) [inline packages/enmeshed_runtime_bridge/integration_test/services/facades/files_facade_test.dart:50]: we should check the expiry in the results when we set it.

**Review by jkoenig134 — CHANGES_REQUESTED** [review]

File: packages/enmeshed_runtime_bridge/integration_test/suite_test.dart [hunk 4]
```diff
@@ -1,8 +1,12 @@
+@Timeout(Duration(minutes: 1))
```
[the remark this answers is not in the document]
Comment (aince42) [reply packages/enmeshed_runtime_bridge/integration_test/suite_test.dart:1]: The default value is 30 seconds, so we increased it to 1 minute

[the remark this answers is not in the document]
Comment (jkoenig134) [reply packages/enmeshed_runtime_bridge/integration_test/suite_test.dart:1]: Ah, I got it. It's per test.

**Review by jkoenig134 — DISMISSED** [review]

**Review by jkoenig134 — CHANGES_REQUESTED** [review]

**Review by jkoenig134 — APPROVED** [review]

**Pull request merged**